date-time-picker/src/androidLibInstrumentedTest/kotlin/AndroidDateTimePickerPresenterTest.kt

```diff
@@ -241,14 +255,16 @@ class AndroidDateTimePickerPresenterTest : DateTimePickerPresenterTests() {
     }
 
     @Test
-    fun testDateBuilderFromActivity() {
-        MainScope().launch { activity?.showDatePicker() }
+    fun testDateBuilderFromActivity() = runBlocking {
```

Comment:
is `runBlocking(Dispatchers.Main)` problematic somehow?

Reply:
seems to hang the test